transmart-data/test_studies/CSR/i2b2metadata/dimension_description.tsv

```diff
@@ -0,0 +1,7 @@
+id	density	modifier_code	value_type	name	packable	size_cd
+0				study		
+1				concept		
+2				patient		
+3	DENSE	CSR_DIAGNOSIS_MOD	T	Diagnosis ID	NOT_PACKABLE	SMALL
```

Comment:
I would name this dimension `Diagnosis` instead of `Diagnosis ID`

Reply:
I see. We have "<ORG> Diagnosis ID" in the customer DB. I guess it's named that way to stress that it contains diagnosis per patient (Ids) as opposed to actual diagnosis names.
My consideration here was to have a dimension name that contains space.